docs/layouts/shortcodes/build_table.html

```diff
@@ -0,0 +1,35 @@
+<style type="text/css">
+  .build-table  {border-collapse:collapse;border-spacing:0;}
+  .build-table th{font-weight: bold;}
```

Comment:
I'm a little surprised that the font-weight for `th` isn't already bold?

Reply:
You're correct!